Nos/Models/CoreData/Event+Hydration.swift

```diff
@@ -91,7 +91,7 @@ extension Event {
         }
         
         var newFollows = Set<Follow>()
-        for jsonTag in jsonEvent.tags {
+        for jsonTag in jsonEvent.tags where jsonTag[safe: 0] == "p" {
```

Comment:
Genius! I looked briefly and couldn't think of a quick way to reduce cyclomatic complexity.